components/brave_wallet/browser/zcash/rust/BUILD.gn

```diff
@@ -45,6 +45,62 @@ source_set("orchard_impl") {
   ]
 }
 
+rust_static_library("zcash_protocol") {
+  visibility = [ ":zcash_primitives" ]
+  crate_name = "zcash_protocol"
+  crate_root = "librustzcash/components/zcash_protocol/src/lib.rs"
+  allow_unsafe = true
```

Comment:
are you sure this is needed here? It should normally only be required for cxx

Reply:
Removed allow_unsafe for zcash_primitives and zcash_protocol, but it is needed for zcash_encoding.
See https://github.com/brave/brave-core/pull/23892#discussion_r1621099945